lncfg/protocol_legacy_on.go

```diff
@@ -20,6 +20,10 @@ type ProtocolOptions struct {
 	// Anchors should be set if we want to support opening or accepting
 	// channels having the anchor commitment type.
 	Anchors bool `long:"anchors" description:"EXPERIMENTAL: enable experimental support for anchor commitments, won't work with watchtowers"`
+
+	// MiniChans should be set if we want to disable support for wumbo
+	// channels, which is the opposite of mini.
+	MiniChans bool `long:"minichannels" description:"force node to not advertise support for wumbo channels"`
```

Comment:
discussed offline: current thinking is wumbo should be _opt-in_ rather than opt-out, since as is nodes will become wumbo-enabled without any config changes. the subset of nodes requiring wumbo is much smaller than that which does not need it, and we want to optimize for the safer default.